frida_tools/_magics.py

```diff
@@ -0,0 +1,111 @@
+import abc
+import json
+
```

Comment:
Shouldn't there be two blank lines here according to PEP-8? (I might me misremembering but at least this is the convention I've used — so hope I didn't get it wrong 😅)